next.config.js

```diff
@@ -29,11 +29,13 @@ module.exports = withTM({
 	images: {
 		domains: ['assets.viaa.be'],
 	},
+	productionBrowserSourceMaps: process.env.DEBUG_TOOLS === 'true',
```

Comment:
Dit gaat nu dus wel werken omdat die hieronder gedefinieerd staat?

Reply:
Will be provided by Tina: https://meemoo.atlassian.net/browse/ARC-484